matrix.yml

```diff
@@ -103,9 +103,9 @@ include:
   - {features: [{<<: *cuda_curr_max, <<: *cccl_cuda_opts}, { name: "cl", version: "14.27" }]} # MSVC 2019
   - {features: [{<<: *cuda_curr_max, <<: *cccl_cuda_opts}, { name: "cl", version: "14.28" }]} # MSVC 2019
   - {features: [{<<: *cuda_curr_max, <<: *cccl_cuda_opts}, { name: "cl", version: "14.29" }]} # MSVC 2019
-  - {features: [{<<: *cuda_curr_max, <<: *cccl_cuda_opts}, { name: "cl", version: "14.34" }]} # MSVC 2022
```

Comment:
Why is this removing 19.34 here but leaving it in below?

Reply:
Like in the rest of the files? Deleting them from the other files would prevent us from building it locally... If we needed to.

I'm fine with removal.

Reply:
I would just leave it in. We do have most other versions still